experiments/generic/pod-cpu-hog.yml

```diff
@@ -0,0 +1,42 @@
+- hosts: localhost
+  vars:
+    a_ns: litmus
+    a_kind: deployment
+    name: engine-nginx5
+    a_label: run=nginx
+    e_name: pod-cpu-hog
+
+  tasks: 
+       
+    - name: Installing pod-cpu-hog Experiment
+      shell: kubectl create -f https://hub.litmuschaos.io/api/chaos?file=charts/generic/pod-cpu-hog/experiment.yaml -n {{ a_ns}}
+
+    - template:
+        src: ../../utils/chaosengine.j2
+        dest: ../../utils/chaosengine.yml
+
+    - name: Create chaos-engine
+      shell: kubectl create -f ../../utils/chaosengine.yml
+
+    - name: Waiting for the chaos complition
+      shell: kubectl get po {{ name }}-runner -n {{ a_ns }} -o custom-columns=:.status.phase --no-headers
+      register: result
+      until: "'Succeeded' in result.stdout"
+      delay: 10
+      retries: 50
+
+    - name: Checking the verdict of the experiment 
+      shell: kubectl get chaosresult {{ name }}-{{ e_name }} -n {{ a_ns }} -o jsonpath='{.spec.experimentstatus.verdict}'
+      register: cr
+
+    - name: Result of verdict test
+      fail:
```

Comment:
does it return with rc != 0

Reply:
if experiment fails (verdict fails) - gitlab job also should fail

Reply:
If any tasks of ansible get fail then thee jobs will automatically fail

Reply:
cool!